public/index.html

```diff
@@ -39,12 +39,12 @@ <h2 class="text-lg md:text-2xl text-green-50 italic animate__animated animate__z
             </div>
             <div class="flex flex-col md:flex-1 items-center justify-center h-screen">
                 <div class="w-[300px] md:w-[400px] p-15">
-                <h1 class="text-4xl font-bold my-5 animate__animated animate__zoomIn">About Me</h1>
-                <p class="text-justify text-2xl text-green-50 italic animate__animated animate__zoomIn">Hello,I'm Adhithya Rajan,a passionate front-end developer and writer. 
-                    I bring stories to life through code,creating captivating web experiences.</p>
-                <p class="text-justify text-2xl py-2 text-green-50 italic animate__animated animate__zoomIn">When I'm not coding,I'm either lost in a book,exploring new music,or simply savoring life's moments.</p>
+                <h1 class="text-4xl font-bold my-3 md:my-5 animate__animated animate__zoomIn">About Me</h1>
```

Comment:
@preveenraj I meant this "my" value. i will share a screenshot.
![scnshot](https://github.com/Adhithyarajan/my-portfolio/assets/99787623/6fd31bd8-fc78-424b-b542-72f54032b236)